core/src/tooltip/tooltip.tsx

```diff
@@ -5,8 +5,18 @@ import { Paragraph } from "../text/text";
 import s from "./tooltip.module.css";
 
 export interface TooltipProps {
+	/**
+	 * The content of the tooltip.
+	 * It could be either a `ReactNode` or function which returns a `ReactNode`
+	 */
 	content: React.ReactNode;
+	/**
+	 * The element to be wrapped
+	 */
 	children: TippyProps["children"];
+	/**
+	 * Placement position
+	 */
```

Comment:
Should have a mention and a link to https://atomiks.github.io/tippyjs/v6/all-props/#placement